.dev/sass/layouts/_header.scss

```diff
@@ -68,7 +68,7 @@ body.no-max-width .site-header-wrapper {
 		position: relative;
 		clear: both;
 
-		h1 {
+		.page-title {
```

Comment:
@EvanHerman This change here is worrisome because any custom CSS people are using to target the page title will break. Any other ideas?